version/version.go

```diff
@@ -0,0 +1,16 @@
+package version
+
+// Ver holds the version derived from the latest git tag
+// Populated using:
+//    go build -ldflags "-X github.com/prebid/prebid-cache/version.Ver=`git describe --tags | sed 's/^v//`"
+// Populated automatically at build / releases in the Docker image
+var Ver string
+
+// VerUnknown is the version used if Ver has not been set by ldflags.
+const VerUnknown = "unknown"
```

Comment:
Please remove `VerUnknown`. It is not used.

Reply:
Good catch. Removed.